server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/util/TestClassWithDatabase.java

```diff
@@ -108,4 +139,26 @@ protected static JdbcDatabaseDelegate setupContainerStartPostgres(PostgreSQLCont
 
     return new JdbcDatabaseDelegate(postgres, "");
   }
+
+  /**
+   * Create and return a hibernate properties object for use in testing functionality that
+   * interacts with the database.
+   * @param postgres Postgres test container representing a test version of the database.
+   * @return Hibernate persistence properties.
+   */
+  public static Properties createHibernateProperties(PostgreSQLContainer<?> postgres) {
+    Properties hibernateProperties = new Properties();
+    hibernateProperties.setProperty("hibernate.driver", "org.postgresql.Driver");
+    hibernateProperties.setProperty("hibernate.url", postgres.getJdbcUrl());
+    hibernateProperties.setProperty("hibernate.user", postgres.getUsername());
```

Comment:
it's so much nicer that this is loaded from a config dynamically instead of my hardcoded version!